x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx

```diff
@@ -165,15 +247,17 @@ export class MapsAppView extends React.Component {
     });
 
     // sync globalState
-    const updatedGlobalState = { filters: filterManager.getGlobalFilters() };
+    const updatedGlobalState: MapsGlobalState = {
+      filters: filterManager.getGlobalFilters(),
+    };
     if (time) {
       updatedGlobalState.time = time;
     }
     updateGlobalState(updatedGlobalState, !this.state.initialized);
   };
 
   _initMapAndLayerSettings() {
-    const globalState = getGlobalState();
+    const globalState: QueryState = getGlobalState();
```

Comment:
Type globalState as `MapsGlobalState` instead of `QueryState`

Reply:
Fixed this.